server/common/fbs/fbs_coders.py

```diff
@@ -13,12 +13,49 @@
 import server.common.fbs.NetEncoding.DictEncoded16FBArray as DictEncoded16FBArray
 import server.common.fbs.NetEncoding.DictEncoded32FBArray as DictEncoded32FBArray
 import server.common.fbs.NetEncoding.DictEncoded8FBArray as DictEncoded8FBArray
+import server.common.fbs.NetEncoding.Int16EncodedXFBArray as Int16EncodedXFBArray
+
+DEFAULT_NUM_BINS = 5000
+
+
+class DenseNumericIntCoder:
+    n_slots = 4
+
+    def encode_array(self, array, builder, _dtype, num_bins=None):
+        if num_bins is None:
+            num_bins = DEFAULT_NUM_BINS
```

Comment:
is there ever a case where is this gets called without an explicit `num_bins` arg? Seems safer to error if so and force callers to specify the bin count.